public/js/services/storage.js

```diff
@@ -0,0 +1,17 @@
+
+angular.module('mean.system')
+  .factory('LocalStorage', ['localStorageService', function(localStorageService) {
+    return {
+      storeToken: function(key, value) {
```

Comment:
Please make the methods here more generic. We dont want to store only token information. You can remove the `Token` from the name